source/src/PascalCompiler/Actions/ActionContainer.hpp

```diff
@@ -24,8 +24,8 @@ auto ActionContainer = boost::hana::make_tuple(
     boost::hana::make_pair(boost::hana::int_c<EnumTypeConstruction>,      boost::hana::type_c<PascalCompiler::EnumTypeConstruction>),
     boost::hana::make_pair(boost::hana::int_c<IndexTypeIDCase>,           boost::hana::type_c<PascalCompiler::IndexTypeIDCase>),
     boost::hana::make_pair(boost::hana::int_c<IndexTypeCXSize>,           boost::hana::type_c<PascalCompiler::IndexTypeCXSize>),
-    boost::hana::make_pair(boost::hana::int_c<Massive1AppendAtStart>,     boost::hana::type_c<PascalCompiler::Massive1AppendAtStart>),
-    boost::hana::make_pair(boost::hana::int_c<Massive1InitVec>,           boost::hana::type_c<PascalCompiler::Massive1InitVec>),
+    boost::hana::make_pair(boost::hana::int_c<IndexListAppendAtStart>,    boost::hana::type_c<PascalCompiler::IndexListAppendAtStart>),
+    boost::hana::make_pair(boost::hana::int_c<IndexListInit>,             boost::hana::type_c<PascalCompiler::IndexListInit>),
```

Comment:
You only renamed enum value, but forgot to rename appropriate action class. Same goes to line before.